Header header
float32 maxAcceleration
float32 maxVelocity
int32 list_size
Waypoint[] wp_list